charmcraft/templates/init/README.md.j2

```diff
@@ -9,17 +9,15 @@ TODO: Describe your charm in a few paragraphs of Markdown
 TODO: Provide high-level usage, such as required config or relations
 
 
-## Developing
+## Relations
```

Comment:
How about just a list of supported relations here (names of charms or relation names) and a boilerplate linked suggestion to see `INTEGRATION.md` for details on relations, their usage and interfaces (data that is exchanged over the relations) ? If this is agreeable then maybe we need a template for `INTEGRATION.md` too.

Reply:
List of relations and the rationale of what they accomplish. We should not ask an end user to go read dev docu to find out what `prometheus_scrape` is for.

Reply:
Agreed. Perhaps then link to INTEGRATION.md from CONTRIBUTING.md ?

Reply:
As with the other comment, I agree that this would be great. There's the old layer index (https://github.com/juju/layer-index), but I haven't seen a list of relation names anywhere, and Google didn't find one. This would be ideal to have once it exists, to be sure.

The same for `INTEGRATION.md`. Once we have a solid template, make it generic and add it here.